src/_modules/Commands.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.8.13 <0.9.0;
 
-import {vulcan} from "./Vulcan.sol";
+import {vulcan, Hevm} from "./Vulcan.sol";
```

Comment:
Use a file-level CommandResult instead of FfiResult.

Reply:
Done